packages/office-addin-dev-settings/src/dev-settings.ts

```diff
@@ -75,10 +80,10 @@ export async function disableRuntimeLogging(): Promise<void> {
   }
 }
 
-export async function enableDebugging(addinId: string, enable: boolean = true, method: DebuggingMethod = DebuggingMethod.Direct): Promise<void> {
+export async function enableDebugging(addinId: string, enable: boolean = true, method: DebuggingMethod = DebuggingMethod.Direct, ieDebug: boolean = false): Promise<void> {
```

Comment:
The idDebug: boolean parameter here should be more like `webview ?: WebViewType` so it would be `undefined` by default (no specific choice) or WebViewType.IE11 or WebViewType.Edge.